README.md

```diff
@@ -116,14 +116,14 @@ use std::net::UdpSocket;
 use std::str::from_utf8;
 
 // Mock a UDP server listening on port 35642. Note that the mock will only listen on the local interface.
-let udp_server_mocker = UdpServerMocker::new_with_port(35642).unwrap();
+let server = UdpServerMocker::new_with_port(35642).unwrap();
 
-// Create the UDP client to test
-let client_socket = UdpSocket::bind("127.0.0.1:34254").unwrap();
-client_socket.connect("127.0.0.1:35642").unwrap();
+// Create the UDP client to test at a random port
+let client_socket = UdpSocket::bind("127.0.0.1:0").unwrap();
```

Comment:
Yes ok good idea, the user immediately knows this possibility 👍

Reply:
I was mostly trying to avoid multiple tests accidentally running at the same time, and failing because one of them already using that port. Still hopeful we can figure out why it doesn't pass tests sometimes